packages/editor/src/client/panel/articleMetadataPanel.tsx

```diff
@@ -131,6 +137,10 @@ export function ArticleMetadataPanel({value, onClose, onChange}: ArticleMetadata
                     onChange?.({...value, socialMediaTitle})
                   }}
                 />
+                <div style={{textAlign: 'right'}}> {`${t("articleEditor.panels.charCount")} ${value.socialMediaTitle?.length}`} </div>
+                { value.socialMediaTitle!.length > socialMediaTitleMax && (
+                <div style={{textAlign: 'right', color: 'gold'}}>{`${t("articleEditor.panels.charCountWarning")} ${socialMediaTitleMax}`}</div>
```

Comment:
Maybe instead of a div, it would be better to set the text corresponding to form input with one of the rsuite form labels or warnings.